ci-operator/config/openshift/logging-view-plugin/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - jgbernalp
+  - kyoto
+approvers:
```

Comment:
You should consider adding @alanconway @periklis and myself to the approver list